frontend/src/Components/Form.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { user } from "../Reducers/user";
+import {
+  StyledForm,
+  Label,
+  Input,
+  H1,
+  Wrapper,
+  FormParagraph,
+  HR,
+  P,
+  ErrorText,
+  SecretText,
+} from "styles/Styles";
+
+import { Button } from "./Button";
+
+export const Form = ({ labelHeading, labelText }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const accessToken = useSelector(store => store.user.login.accessToken);
+  const userId = useSelector(store => store.user.login.userId);
+  const secretMessage = useSelector(store => store.user.login.secretMessage);
+  const errorMessage = useSelector(store => store.user.login.errorMessage);
+  const dispatch = useDispatch();
+
+  const createUser = event => {
+    event.preventDefault();
+    // const USER_URL = "http://localhost:8080/users";
+    const USER_URL = "https://user-authorisation.herokuapp.com/users";
+
+    fetch(USER_URL, {
+      method: "POST",
+      body: JSON.stringify({ username, password }),
+      headers: { "Content-Type": "application/json" },
+    })
+      .then(res => {
+        if (res.ok) {
+          return res.json();
+        } //eslint-disable-next-line
+        throw "Could not create user"; // we want to display error message from backend here
+      })
+      .then(json => {
+        dispatch(
+          user.actions.setAccessToken({ accessToken: json.accessToken })
+        );
+        dispatch(user.actions.setUserId({ userId: json.userId }));
+        localStorage.setItem("accessToken", json.accessToken);
```

Comment:
We are a bit unsure what this is suppose to do, if we create an account, and reload the page we are thrown out to the sign in page again. 

If you want to store accessToken it should be stated in the initialState `accessToken: localStorage.accessToken || null,` and setItem in your reducer setAccessToken` localStorage.setItem('accessToken', accessToken)`, the same goes for userId

Reply:
@thereselinden  is right here. Please fix it because you can't let your users be restricted after the refresh. 🤓